src/Routers/ClassesRouter.js

```diff
@@ -180,7 +180,7 @@ export class ClassesRouter extends PromiseRouter {
     if (body.limit || body.limit === 0) {
       options.limit = Number(body.limit);
     } else {
-      options.limit = Number(100);
+      options.limit = Number(defaultLimit || 100);
```

Comment:
The default value should be specified in the server option definition through the `:DEFAULT:` attribute. If the option is not set, the parameter `req.config.defaultLimit` then is `100` by default.

Reply:
@vzukanov I think this is the last minor change needed, then we can merge.